src/Bicep.Cli/Program.cs

```diff
@@ -37,6 +39,10 @@ public Program(IResourceTypeProvider resourceTypeProvider, TextWriter outputWrit
 
         public static int Main(string[] args)
         {
+            string profilePath = Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.LocalApplicationData), LanguageConstants.LanguageId);
+            Directory.CreateDirectory(profilePath);
+            ProfileOptimization.SetProfileRoot(profilePath);
```

Comment:
>SetProfileRoot [](http://example.com/codeflow?start=32&length=14)

How do we deal with clean up of the data? With local app data, we could only do cleanup in the windows setup. Other install methods couldn't clean up after themselves.

Could we use a Temp path instead? There would be a risk of losing the data, but it can reconstruct it the next time, so shouldn't be any impact.

Reply:
Using current user temp folder and `.bicep` subfolder within.